bin/uglifyjs

```diff
@@ -352,3 +374,16 @@ function to_cache(key) {
     }
     return cache[key];
 }
+
+function skip_key(key) {
+    switch (key) {
+      case "cname":
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/1879/commits/4116731b5a8c24c2926130f25a60894ff1ff2428